src/RecordTask.cc

```diff
@@ -1071,6 +1071,7 @@ void RecordTask::emulate_SIGCONT() {
   for (Task* t : thread_group()->task_set()) {
     auto rt = static_cast<RecordTask*>(t);
     LOG(debug) << "setting " << tid << " to NOT_STOPPED due to SIGCONT";
+    rt->clear_stashed_group_stop();
```

Comment:
Would be nice if this had a test.

Reply:
It kinda does. exec_stopsig tests this, but it appears to depend on how quick the signal gets delivered.